handler/oauth2/flow_authorize_jwt_token.go

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright © 2015-2018 Aeneas Rekkas <aeneas+oss@aeneas.io>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author		Aeneas Rekkas <aeneas+oss@aeneas.io>
+ * @copyright 	2015-2018 Aeneas Rekkas <aeneas+oss@aeneas.io>
+ * @license 	Apache-2.0
+ *
+ */
+
+package oauth2
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ory/fosite"
+	"github.com/ory/x/errorsx"
+	"gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+const grantTypeJwtBearer = "urn:ietf:params:oauth:grant-type:jwt-bearer"
+
+type AuthorizeJwtGrantHandler struct {
+	AuthorizeJwtGrantStorage AuthorizeJwtGrantStorage
```

Comment:
Idiomatic Go upper-cases abbreviations:

```suggestion
	AuthorizeJWTGrantStorage AuthorizeJWTGrantStorage
```

Reply:
will change